OpenRA.Mods.Common/Traits/World/Locomotor.cs

```diff
@@ -459,26 +459,17 @@ void UpdateCellBlocking(CPos cell)
 			using (new PerfSample("locomotor_cache"))
 			{
 				var cache = blockingCache[cell.Layer];
-
-				var actors = actorMap.GetActorsAt(cell);
 				var cellFlag = CellFlag.HasFreeSpace;
-
-				if (!actors.Any())
-				{
-					cache[cell] = new CellCache(default, cellFlag);
```

Comment:
Is it ok to not write to the cache if we don't have actors here? (It seems this line got entirely removed.)

Reply:
If there are no actors, then the loop over the actors does nothing and we hit `cache[cell] = new CellCache(cellImmovablePlayers, cellFlag, cellCrushablePlayers);` after the loop to set the cache.